src/main/java/hlf/java/rest/client/controller/FabricOperationsController.java

```diff
@@ -126,4 +126,21 @@ public ResponseEntity<ClientResponseModel> getDeserializedJson(
       @RequestParam(name = "prettyPrint", required = true) boolean prettyPrint) {
     return serializationUtil.decodeContents(encodedJson, decodeInterior, prettyPrint);
   }
+
+  /**
+   * Add anchor peer(s) of an organization to a channel. Anchor peer addition should be done once
+   * the peer nodes of the organization have joined the channel.
+   *
+   * @param channelName - the name of the channel for which you wish to add the anchor peer nodes
+   *     to.
+   * @param anchorPeerParamsDTO - contains the details for the organization peers you wish to be
+   *     added to the channel as anchor peer.
+   * @return ResponseEntity<ClientResponseModel> - contains the result of the operation.
+   */
+  @PostMapping(value = "/channel/{channelName}/add_anchor_peer")
+  public ResponseEntity<ClientResponseModel> addAnchorPeersToChannel(
```

Comment:
Thanks, I found this from the official Fabric documentation for anchor peer addition -`Because we are updating a section of the channel configuration that only affects Org1, other channel members do not need to approve the channel update`. Do we still see a possibility ? I could add a TODO comment.

Reply:
Thank you @nidhi-singh02 . We could skip the comment.